images/scripts/arch.setup

```diff
@@ -182,7 +182,7 @@ pacman -Syu --noconfirm
 pacman -Scc --noconfirm
 
 # Reduce image size, remove cloud-init
-pacman -Rscn --noconfirm cloud-init
+pacman -R --noconfirm cloud-init
```

Comment:
This is fine -- This is primarily about being able to boot our VM without a cloud-init info volume (to avoid boot hangs).